### PR TITLE
Add tar zip

### PR DESCRIPTION
**Pull request by at88mph**

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -10,12 +10,13 @@ jobs:
...
-    - uses: actions/checkout@v2
-    - name: Set up JDK 1.8
-      uses: actions/setup-java@v1
+    - uses: actions/checkout@v3
+    - name: Set up JDK 11
+      uses: actions/setup-java@v3
       with:
-        java-version: 1.8
-    
+        distribution: 'temurin'
```
Comment (andamian) [inline .github/workflows/gradle.yml:17]: Is that required?

**Comment by at88mph** [comment]
CADC-13714

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (at88mph) [reply .github/workflows/gradle.yml]: Yes.  Declaring the `distribution` is mandatory either the `setup-java@3`.  We also require JDK 1.8 as `downloadManager` still runs on that on CADC infrastructure.

File: cadc-download-manager-server/src/main/java/ca/nrc/cadc/dlm/server/PackageServlet.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        });
+    }
+
+    private Map<String, Object> getPayload(final HttpServletRequest request) {
+        final Map<String, Object> payload = new HashMap<>();
+        final Map<String, String[]> requestParameters = request.getParameterMap();
+
+        final String[] publisherIDs = requestParameters.get(PackageServlet.REQUEST_PARAM_URI_KEY);
+        if (publisherIDs == null || publisherIDs.length == 0) {
+            throw new IllegalArgumentException("Nothing specified to download.  Use tuple=<URI>.");
```
Comment (andamian) [inline cadc-download-manager-server/src/main/java/ca/nrc/cadc/dlm/server/PackageServlet.java:89]: Minor comment: could replace `tuple` with `PackageServlet.REQUEST_PARAM_URI_KEY`. It applies to the other ones too.

File: cadc-download-manager/build.gradle [hunk 3]
```diff
@@ -24,13 +24,12 @@ dependencies {
     implementation 'org.opencadc:cadc-util:[1.6,)'
     implementation 'org.opencadc:cadc-registry:[1.1.1,)'
     implementation 'org.opencadc:cadc-app-kit:[1.0,)'
-    implementation 'org.opencadc:cadc-vos:[1.0,)'
+    implementation 'org.opencadc:cadc-vos:1.2.3'
```
Comment (andamian) [inline cadc-download-manager/build.gradle:27]: Pinned on purpose?

File: cadc-upload-manager/build.gradle [hunk 4]
```diff
@@ -24,12 +24,12 @@ dependencies {
     implementation 'org.opencadc:cadc-registry:[1.5,2.0)'
     implementation 'org.opencadc:cadc-util:[1.6,2.0)'
     implementation 'org.opencadc:cadc-uws:[1.0,2.0)'
-    implementation 'org.opencadc:cadc-vos:[1.2.2,2.0)'
+    implementation 'org.opencadc:cadc-vos:1.2.3'
```
Comment (andamian) [inline cadc-upload-manager/build.gradle:27]: Same here.

**Review by andamian — APPROVED** [review]

File: cadc-download-manager/build.gradle (hunk 3 above)
Comment (at88mph) [reply cadc-download-manager/build.gradle:27]: Yes, due to the JDK 1.8 requirement.

File: cadc-upload-manager/build.gradle (hunk 4 above)
Comment (at88mph) [reply cadc-upload-manager/build.gradle:27]: Yes, due to the JDK 1.8 requirement.

File: cadc-download-manager/build.gradle (hunk 3 above)
Comment (andamian) [reply cadc-download-manager/build.gradle:27]: I suggest you add a comment so we remember.

**Comment by at88mph** [comment]
Documented locked `cadc-vos` version.  Thanks, @andamian .

**Pull request merged**